plugins/english/faqwikius.ts

```diff
@@ -0,0 +1,120 @@
+import { Plugin } from "@typings/plugin";
+import { fetchApi, fetchFile } from "@libs/fetch";
+import { NovelStatus } from "@libs/novelStatus";
+import { load as parseHTML } from "cheerio";
+
+class FaqWikiUs implements Plugin.PluginBase {
+  id = "FWK.US";
+  name = "Faq Wiki";
+  site = "https://faqwiki.us/";
+  version = "1.0.0";
+  icon = "src/en/faqwikius/icon.png";
+
+  async popularNovels(
+    page: number,
+    { showLatestNovels }: Plugin.PopularNovelsOptions,
+  ): Promise<Plugin.NovelItem[]> {
+    const sort = showLatestNovels
+      ? ""
+      : "";
+
+    const body = await fetchApi(this.site + sort).then((res) => res.text());
+    const loadedCheerio = parseHTML(body);
+
+    const novels: Plugin.NovelItem[] = loadedCheerio(".wp-block-table > table > tbody > tr")
+      .map((index, element) => {
+        const name = loadedCheerio(element).find("strong").first().text();
+        let cover = loadedCheerio(element).find("img").attr("data-ezsrc"); 
+
+        // Remove the appended query string 
+        if (cover) {
+             cover = cover.replace(/\?ezimgfmt=.*$/, ''); // Regular expression magic!
+        }
+
+        const path = "" + loadedCheerio(element).find("a").attr("href");
```

Comment:
![image](https://github.com/LNReader/lnreader-sources/assets/86464880/ec2f1233-417e-496c-a1da-eea7b2c597f0)
Pls remove the domain in novel's path

Reply:
Done https://github.com/LNReader/lnreader-sources/pull/792/commits/6fb6a355b3b28964e25b3991819a922d006b322e